pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -1075,6 +1075,7 @@ func getInfinispanConditions(pods []corev1.Pod, m *infinispanv1.Infinispan, prot
 			if kube.IsPodReady(pod) {
 				members, err := cluster.GetClusterMembers(pod.Name)
 				if err == nil {
+					sort.Strings(members)
```

Comment:
Why is the sort required?

Reply:
The code glues comma separated nodes from each pod. Probably it's too redundant but we should know that keys are always the same.